usaspending_api/awards/models.py

```diff
@@ -291,6 +305,7 @@ class FinancialAssistanceAward(AwardAction):
     reporting_period_start = models.DateField(blank=True, null=True)
     reporting_period_end = models.DateField(blank=True, null=True)
     last_modified_date = models.DateField(blank=True, null=True)
+    submitted_type = models.CharField(max_length=1, blank=True, null=True, verbose_name="Submitted Type")
```

Comment:
@afrasier @bsweger Yes possible domain values for SubmittedType are 0 for CSV or 1 for txt, file type.